### DISCUSSION TITLE
Need help fix this issue (DNS record)

### DISCUSSION THREAD

**Discussion by DanteAnuj** [postalserver/postal#2256] [Help with using Postal]
I need help setting DNS for postal, idk what I am doing wrong I tried everything but can't find it. Please can someone help me with this? Much appreciated.

SS - ![download](https://user-images.githubusercontent.com/58907150/204231674-da5cc325-9c65-48cb-a82d-5817d3d18b7a.png)

**Comment by willpower232** [comment]
there is supposed to be an MX record in the return path section but that does not appear to be in the documentation so that is a little awkward, we have

`rp.postal.ourdomain.com MX 10 mx.postal.ourdomain.com`

hope that helps

**Comment by DanteAnuj** [comment]
this is the MX record I have set on the DNS. what I need to update is sorry if I am being dumb but I am not pretty good at this.

![Screenshot_1](https://user-images.githubusercontent.com/58907150/204238026-a5e26589-683a-4f9b-997a-7ee3a70965dd.png)

**Reply by willpower232** [reply]
and is psrp.yourdomain.com a CNAME to rp.mail.team-billingz.com?

**Comment by DanteAnuj** [comment]
I am sharing the full screenshot it will be easy then
![Screenshot_2](https://user-images.githubusercontent.com/58907150/204250723-51d7ef5a-379d-473e-9f83-c4e7ed142396.png)

**Reply by willpower232** [reply]
it looks like you're missing a bit

![image](https://user-images.githubusercontent.com/1619102/204258746-deeb039d-2824-49e0-89f0-e446a416c4de.png)

**Comment by DanteAnuj** [comment]
Thank you so much for the help sir, really appreciate that.  I need a little more help sir

Still, to achieve a 10/10 score on Mail tester these two issues still popping up right now. Maybe I am doing something strong

![Screenshot_4](https://user-images.githubusercontent.com/58907150/204259962-8b471374-abb1-42ca-a3d5-be01e67025c9.png)

![Screenshot_3](https://user-images.githubusercontent.com/58907150/204259915-7786893e-2f25-4927-909d-6a132faad8cc.png)

**Reply by willpower232** [reply]
Looking back at your screenshot, are you still missing the bit from the rp.mail and spf.mail spf TXT records?

Regarding your reverse DNS, you will need to arrange that with your server provider. It may be as easy as naming your server a certain thing in the control panel or you might have to contact them to set it up properly.

**Comment by DanteAnuj** [comment]
Thank you for your help sir I have fixed the record issues. Regarding the DNS what do I mail the team ? to set my rDNS in the domain?

**Reply by willpower232** [reply]
reverse DNS is set by whoever manages the IP address on the server so you will likely need to ask your server provider to set it as whichever domain name you would like

**Comment by DanteAnuj** [comment]
so I have to talk to my VPS provider to set rDNS.

**Reply by willpower232** [reply]
yep
